docs/features/multiple-networks/multi-homing.md

```diff
@@ -166,71 +221,21 @@ localnet network.
   to the pods.
 - `vlanID` (integer, optional): assign VLAN tag. Defaults to none.
 - `allowPersistentIPs` (boolean, optional): persist the OVN Kubernetes assigned
-   IP addresses in a `ipamclaims.k8s.cni.cncf.io` object. This IP addresses will
-   be reused by other pods if requested. Useful for KubeVirt VMs. Only makes
-   sense if the `subnets` attribute is also defined.
-- `physicalNetworkName` (string, optional): the name of the physical network to
```

Comment:
@RamLavi please open a follow-up adding the `physicalNetworkName` attribute back.

I think this was the by-product of a rebase gone bad. 

Sorry I didn't catch it before.